src/V3EmitV.cpp

```diff
@@ -563,7 +563,11 @@ class EmitVBaseVisitor VL_NOT_FINAL : public EmitCBaseVisitor {
     }
     virtual void visit(AstBasicDType* nodep) override {
         if (nodep->isSigned()) putfs(nodep, "signed ");
-        putfs(nodep, nodep->prettyName());
+        if (nodep->keyword() == AstBasicDTypeKwd::LOGIC_IMPLICIT) {
+            putfs(nodep, "logic");
+        } else {
+            putfs(nodep, nodep->prettyName());
+        }
```

Comment:
I think prettyName for LOGIC_IMPLICIT should be changed to just "logic" as the user doesn't care about LOGIC_IMPLICIT, I'd also add a comment it needs to be logic to appease V3EmitV.